### DISCUSSION TITLE
E-mail Validation from Docker Images

### DISCUSSION THREAD

**Discussion by privacytime101** [FreshRSS/FreshRSS#5679] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
According [to the docs](https://github.com/FreshRSS/FreshRSS/blob/b82c41f6b99be467dab5c561deefbe01eee50b8d/docs/en/admins/05_Configuring_email_validation.md?plain=1#L22C1-L22C1), it's currently not possible to send SMTP e-mails while using the official docker images. I do have an SMTP server ready to go but apparently it's not supported yet. Maybe I'm just misinterpreting this line?

**Describe the solution you’d like**
I think adding SMTP capabilities to the images would benefit public instances using Docker.

**Comment by Alkarex** [comment, marked as answer]
Everything should work from the Docker image. You just need to be careful about network visibility/routing (for instance, you cannot use `localhost`, but that is the same as anything else in Docker. The easiest is probably to run your SMTP server also in Docker, or on another server).

We indeed do not have a Web interface to set it up, but that can be set up in `./data/config.php` in the `'smtp'` section.
See https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md#modify-the-configuration-of-a-running-freshrss-instance

On the model of:
https://github.com/FreshRSS/FreshRSS/blob/b82c41f6b99be467dab5c561deefbe01eee50b8d/config.default.php#L176-L188

Alternatively, those parameters can be provided prior to installing FreshRSS, see https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md#provide-default-global-settings

Maybe @marienfressinaud who is using the SMTP feature could provide more tips.

**Reply by Frenzie** [reply]
@marienfressinaud wrote a big blog post about how moving away from Docker magically improved performance significantly more than a few percentage points and greatly reduced complexity so I don't know about that. xD

**Comment by privacytime101** [comment]
Thanks for the replies, I'll look into this.
